staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/metrics.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kmsv2
+
+import (
+	"sync"
+	"time"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+// TODO(aramase): prefix metrics to distinguish from other v1 Envelope providers
```

Comment:
should this be added in this PR?

Reply:
I already have the `kmsv2_` prefix for the metrics: https://github.com/kubernetes/kubernetes/pull/111126/files#diff-9d668188d00c3eca92e8c3f107eebfa5d94fabed3af81c02fda88a36efe2a193R64, just need to delete the comment.